libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm

```diff
@@ -34,7 +34,7 @@ BATT_MONITOR	20
 BATT_VOLT_MULT	22.7
 BRD_SAFETYENABLE	0
 BRD_SAFETY_MASK	0
-BRD_SER1_RTSCTS	2
+BRD_SER1_RTSCTS	1
```

Comment:
The BBB uart flow control pins (UART5_CTS/RTS) are also used as SYS_BOOT pins for bootup options. These two pins control the selected external crystal frequency:   
    00b = 19.2 MHz
    01b = 24 MHz
    10b = 25 MHz
    11b = 26 MHz

The correct configuration selected by pull up/down resistors is 01b, 24 MHz. However, with the Cube controlling the BBB_UART5_CTS line, there is a good chance it will drive low at the same time BBB boots up. This would cause the boot pins to be 00b, 19.2 MHz. This mismatch in frequency would mess up timing on the BBB. 

We may need the Cube to delay the driving of it's RTS line as a workaround.

Reply:
@dat-tran-matternet There's an ardupilot parameter change that we plan to test to prevent the Cube from driving those lines for at least the first second. We're tracking that here: https://matternet.atlassian.net/browse/AVX-324

I'll updated the description of that ticket with the detail you shared in this comment.